LEAF_Nexus/scripts/refreshOrgchartEmployees.php

```diff
@@ -31,81 +31,104 @@
 			WHERE userName=:userName";
 
-	$sql2 = "UPDATE employee
-		SET lastName=:lastName,
-		firstName=:firstName,
-		middleName=:midInit,
-		phoneticFirstName=:phoneticFname,
-		phoneticLastName=:phoneticLname,
-		domain=:domain,
-		deleted=:deleted,
-		lastUpdated=:lastUpdated
-		WHERE userName=:userName";
-
-	#used to disable if not found in national
+    $sql2 = "UPDATE employee
+            SET lastName=:lastName,
+            firstName=:firstName,
+            middleName=:midInit,
+            phoneticFirstName=:phoneticFname,
+            phoneticLastName=:phoneticLname,
+            domain=:domain,
+            deleted=:deleted,
+            lastUpdated=:lastUpdated
+            WHERE userName=:userName";
+
+    #used to disable if not found in national
     $sql3 = "UPDATE employee
-        SET deleted=:deleted
-        WHERE userName=:userName";
+            SET deleted=:deleted
+            WHERE userName=:userName";
 
-	$res = $phonedb->prepared_query($sql, $vars);
+    $sql4 = "SELECT userName, deleted, lastUpdated
+             FROM employee
+             WHERE userName=:userName AND lastUpdated > 0";
 
-	if (count($res) == 0){
-	    //if there is no record in nat, disable the account.
-	    $vars = array(
-	        ':userName' => $userName,
+    $res = $phonedb->prepared_query($sql, $vars);
+
+    if (count($res) == 0) {
+        //if there is no record in nat, disable the account.
+        $vars = array(
+            ':userName' => $userName,
             ':deleted' => time()
         );
-	    $db->prepared_query($sql3, $vars);
+        $db->prepared_query($sql3, $vars);
     }
 
-	if (count($res) > 0) {
-		$vars = array(
-				':userName' => $res[0]['userName'],
-				':lastName' => $res[0]['lastName'],
-				':firstName' => $res[0]['firstName'],
-				':midInit' => $res[0]['middleName'],
-				':phoneticFname' => $res[0]['phoneticFirstName'],
-				':phoneticLname' => $res[0]['phoneticLastName'],
-				':domain' => $res[0]['domain'],
-				':deleted' => $res[0]['deleted'],
-				':lastUpdated' => $res[0]['lastUpdated']
-		);
-
-		// sets local employee table
-		$db->prepared_query($sql2, $vars);
-
-		// sets local employee_data table
-		updateEmployeeData($res[0]['empUID'], $empUID);
-	}
+    if (count($res) > 0) {
+        $vars = array(
+            ':userName' => $res[0]['userName'],
+            ':lastName' => $res[0]['lastName'],
+            ':firstName' => $res[0]['firstName'],
+            ':midInit' => $res[0]['middleName'],
+            ':phoneticFname' => $res[0]['phoneticFirstName'],
+            ':phoneticLname' => $res[0]['phoneticLastName'],
+            ':domain' => $res[0]['domain'],
+            ':deleted' => $res[0]['deleted'],
+            ':lastUpdated' => $res[0]['lastUpdated']
+        );
+
+        // sets local employee table
+        $db->prepared_query($sql2, $vars);
+
+        // sets local employee_data table
+        updateEmployeeData($res[0]['empUID'], $empUID);
+    }
+
+
+    if (count($res) > 0) {
```

Comment:
why are there two identical if statements.  Can't everything be handled under one?